packages/teams-js/src/public/storage.ts

```diff
@@ -0,0 +1,24 @@
+import { GlobalVars } from '../internal/globalVars';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { HostClientType } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Contains functionality to allow web apps to store data in webview cache
+ *
+ * @beta
+ */
+export namespace storage {
+  export function isWebStorageClearedOnUserLogOut(): boolean {
+    ensureInitialized();
+    // return true as storage is always cleared on desktop.
+    if (GlobalVars.hostClientType === HostClientType.desktop) {
+      return true;
+    }
+    return isSupported();
+  }
+
+  export function isSupported(): boolean {
```

Comment:
Please add reference documentation comments (you can take a look at other files for isSupported comments as reference)